docs/index.md

```diff
@@ -0,0 +1,115 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The In-App payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
```

Comment:
We aren't super consistent with the naming of the SDK - is In-App Payments SDK the name or just Payments SDK? Either way we should make the capitalization and naming the same across this doc.